### DISCUSSION TITLE
I am Authorizing a App with read email access, Is that app can see my all Email or only primary email?

### DISCUSSION THREAD

**Discussion by vimal-verma** [community/community#23231] [New to GitHub]
<p>I am Authorizing a App (<a href="https://www.netlify.com/" rel="noopener nofollow ugc">netlify</a>) with <strong>Personal user data</strong> <small> Email addresses (read-only)</small>, Is netlify can see my all email or only primary email?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>I believe it’s all of them that they can see, but it’s also possible they don’t actually view or make use of more than one. It depends on how they implement it.</p>
